.clang-format

```diff
@@ -18,15 +21,16 @@ BraceWrapping:
   AfterExternBlock: true
   BeforeCatch: true
   BeforeElse: true
-  IndentBraces: true
   BeforeLambdaBody: true
   BeforeWhile: true
-BreakBeforeBraces: Allman
+  IndentBraces: false
+BreakBeforeBraces: Custom
 BreakBeforeInheritanceComma: true
-BreakConstructorInitializersBeforeComma: true
+BreakConstructorInitializers: BeforeComma
 BreakStringLiterals: false
 ColumnLimit: 0
 ConstructorInitializerAllOnOneLineOrOnePerLine: true
+IndentCaseBlocks: true

```

Comment:
Otherwise `break` is indented to the same level as `case` (or rather, _not_ indented) when that `case` contains a block statement.